internal/npm_install/generate_build_file.js

```diff
@@ -340,6 +340,23 @@ filegroup(
     tags = ["NODE_MODULE_MARKER"],
 )
 
+filegroup(
+    name = "${pkg._dir}__dts",
```

Comment:
we generally call this "typings" instead of "dts"

Reply:
much better thanks